RingPublishingTracking/src/main/java/com/ringpublishing/tracking/RingPublishingTracking+ContentPerformanceMonitor.kt

```diff
@@ -119,13 +124,10 @@ fun RingPublishingTracking.reportContentPageView(
     contentKeepAliveDataSource: RingPublishingTrackingKeepAliveDataSource,
 )
 {
-	// todo: Implement
-// 	reportDelegate.reportContentPageView(
-// 		contentMetadata,
-// 		contentPageViewSource,
-// 		partiallyReloaded,
-// 		contentKeepAliveDataSource
-// 	)
+	configurationManager.updateContentPageViewConfiguration(contentMetadata, currentStructurePath, partiallyReloaded)
```

Comment:
zobacz jak Artur to zrobił bo wydaje mi sie że czytelniej / prościej

Reply:
Pozmieniałem tak jak u Artura, by każda properta była aktualizowana osobną metodą.